lib/livebook_web/live/hub/teams/deployment_group_live.ex

```diff
@@ -114,6 +120,36 @@ defmodule LivebookWeb.Hub.Teams.DeploymentGroupLive do
                     return_to={~p"/hub/#{@hub.id}/deployment-groups/edit/#{@deployment_group.id}"}
                   />
                 </div>
+
+                <div :if={@deployment_group.mode == "online"} class="flex flex-col space-y-4">
```

Comment:
`mode` should be an atom as per the schema, we should align all places where we use "online"/"offline".